### ISSUE TITLE
Last option "swipe gesture" needs to be deleted in G219 Ensuring that a single pointer alternative

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#2404]
In G219 there is an alternative added which is the core of the problem for dragging or pointer gestures... :-)
https://w3c.github.io/wcag/techniques/general/G219

Total text:

> Such dragging movements are difficult or impossible to carry out for some users with motor disabilities. The alternative to dragging movements operates the underlying function by one or several [single pointer](https://w3c.github.io/wcag/guidelines/#dfn-single-pointer) activations. A single tap or click may reveal controls (arrows) to move a target in a stepwise fashion; open a drop-down menu where the drop position can be selected; or allow moving it to an ajacent postion by a swipe gesture.

Last part is:

> or allow moving it to an ajacent postion by a swipe gesture.

The swipe gesture part is a problem for "some users with motor disabilities"...

(also ajacent should be adjacent... but no change needed as all needs to be deleted)

**Comment by detlevhfischer** [comment]
@jake-abma I don't think we can remove swipe from the single pointer definition easily, or even that this is a fight worth fighting at this time. The next step will be to determine whether we want to go ahead with 2.5.7 Dragging Movements at all. I intend to resume serious work on the Understanding text of 2.5.7 when the WG has decided to include it to receive public more comments (but adding your editorial suggestions was a minor effort hence the PR #2401 ).
